docs/root/start/sandboxes/cache.rst

```diff
@@ -0,0 +1,238 @@
+as when a response is validated the ``date`` header is updated, while the body stays the same. Validated responses do not have an ``age`` header.
+
+Running the Sandbox
+~~~~~~~~~~~~~~~~~~~
+
+The following documentation runs through the setup of an Envoy cluster organized
+as is described in the image above.
+
+**Step 1: Install Docker**
+
+Ensure that you have a recent versions of ``docker`` and ``docker-compose`` installed.
+
+A simple way to achieve this is via the `Docker Desktop <https://www.docker.com/products/docker-desktop>`_.
+
+**Step 2: Clone the Envoy repo**
+
+If you have not cloned the Envoy repo, clone it with:
+
+``git clone git@github.com:envoyproxy/envoy``
+
+or
+
+``git clone https://github.com/envoyproxy/envoy.git``
+
+**Step 3: Start all of our containers**
+
+.. code-block:: console
+
+    $ pwd
+    envoy/examples/cache
+    $ docker-compose build --pull
+    $ docker-compose up -d
+    $ docker-compose ps
+
+           Name                      Command            State                             Ports                          
+    ------------------------------------------------------------------------------------------------------------------------
+    cache_front-envoy_1   /docker-entrypoint.sh /bin ... Up      10000/tcp, 0.0.0.0:8000->8000/tcp, 0.0.0.0:8001->8001/tcp
+    cache_service1_1      /bin/sh -c /usr/local/bin/ ... Up      10000/tcp, 8000/tcp                                      
+    cache_service2_1      /bin/sh -c /usr/local/bin/ ... Up      10000/tcp, 8000/tcp    
+
+**Step 4: Test Envoy's HTTP caching capabilities**
+
+You can now send a request to both services via the ``front-envoy``. Note that since the two services have different routes,
+identical requests to different services have different cache entries (i.e. a request sent to service 2 will not be served by a cached
+response produced by service 1).
+
+To send a request: 
+
+``curl -i localhost:8000/service/<service_no>/<response>``
+
+``service_no``: The service to send the request to, 1 or 2.
+
+``response``: The response that is being requested. The responses are found in :repo:`/examples/cache/responses.yaml`.
+
+
+The existing example responses are:
+
+- ``valid-for-minute``
+    This response remains fresh in the cache for a minute. After which, the response gets validated by the backend service before being served from the cache.
+    If found to be updated, the new response is served (and cached). Otherwise, the cached response is served and refreshed.
+
+- ``private``
+    This response is private; it cannot be stored by shared caches (such as proxies). It will always be served from the backend service.
+    
+- ``no-cache``
+    This response has to be validated every time before being served.
+    
+You can change the responses headers and bodies (or add new ones) while the sandbox is running to experiment.
+
+Example responses
+^^^^^^^^^^^^^^^^^^^^^
+1. A response that stays fresh for a minute (valid-for-minute):
```

Comment:
i think using subheaders with eg `valid-for-minute` would make it more obvious which bits of text relate to which example reponse